examples/about_dialog/main.rs

```diff
@@ -0,0 +1,44 @@
+use gtk::gdk;
+use gtk::gio;
+use gtk::glib;
+use gtk::prelude::*;
+
+fn main() {
+    let app = gtk::Application::new(None::<&str>, gtk::gio::ApplicationFlags::empty());
```

Comment:
Please use the builder pattern here. Also, similar to the other examples give it an app id